README.md

```diff
@@ -101,14 +98,14 @@ is compiled differently. A typical example of this would be `conda`-installed Te
 
 
 #### Compatibility Matrix
-*GPU is supported from version `0.2.0`*
+*GPU is supported by version `0.2.0` and later*
 
-| TFRA | TensorFlow | Compiler  | CUDA | CUDNN | Compute Capability |
-|:----------------------- |:---- |:---------| :------------ | :---- | :------------ |
-| 0.3.0 | 2.5.1  | GCC 7.3.1 | 11.2 | 8.1 | 6.0, 6.1, 7.0, 7.5, 8.0, 8.6 |
-| 0.2.0 | 2.4.1  | GCC 7.3.1 | 11.0 | 8.0 | 6.0, 6.1, 7.0, 7.5, 8.0 |
-| 0.2.0 | 1.15.2  | GCC 7.3.1 | 10.0 | 7.6 | 6.0, 6.1, 7.0, 7.5 |
-| 0.1.0 | 2.4.1  | GCC 7.3.1 | - | - | - |
+| TFRA  | TensorFlow | Compiler  | CUDA | CUDNN | Compute Capability |
+|:------|:---- |:---------| :------------ | :---- | :------------ |
+| 0.3   | 2.5.1  | GCC 7.3.1 | 11.2 | 8.1 | 6.0, 6.1, 7.0, 7.5, 8.0, 8.6 |
```

Comment:
Will the patch version number be ignored like: `pip install tensorflow-recommenders-addons==0.3`?